### DISCUSSION TITLE
FR: Option to mark newly added external models as disabled by default

### DISCUSSION THREAD

**Discussion by i0ntempest** [open-webui/open-webui#7706] [Ideas]
# Feature Request

**Is your feature request related to a problem? Please describe.**
When I added my OpenAI API key a bunch of new external models cluttered my model list. Given that OpenAI changes/releases new models over time this will most likely happen again. Rather than disabling models not needed, it's better imo to only enable those needed.

**Describe the solution you'd like**
Add an option one can set to mark newly added external models as disabled by default.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by i0ntempest** [comment]
@tjbck Is there a way to do this currently if I may ask? Or are you just moving all FRs to discussions?

**Comment by toihr** [comment]
Hi wanted to ask if this is still on anyones mind. We want to roll out our own Company Wide Chat Interface but having new not approved Models be automatically added and enabled is a big nono

**Comment by Classic298** [comment]
I don't see the need for this as newly added modles are private by default and thus only an administrator could see them.

Why is this an issue at all whether it's enabled or disabled if it's private by default

**Comment by i0ntempest** [comment]
It was (and still is) an issue because I do not want new models to appear unannounced in my model list, regardless of the account type. Some models (embeddings, image gen etc) should not appear in the chat model list anyways.

**Reply by Classic298** [reply]
@i0ntempest why not use the allowlist then? there is a model allowlist in the connection settings why do you not make use of that then
